### DISCUSSION TITLE
Sveltekit multilang demo

### DISCUSSION THREAD

**Discussion by pavelstianko** [ivanhofer/typesafe-i18n#630] [Show and tell]
Hello and thank you for this amazing library! We tried to create the multilang Sveltekit demo with some routing rules:
- use lang-codes in url for locale (/de, /cs)
- hide default language lang-code in url (en is just /)
- detect locale from browser (if it is not set by url or cookie)
- if you set the language, save the locale to cookie for next visit

There is the demo: https://lang.bitworks-dev.cz/

**Comment by ivanhofer** [comment]
Looks good. Thanks for sharing!

**Comment by ansbr** [comment]
@pavelstianko Can you share your solution? Do you have this solution in a public repository? I want make the same for my website.

**Reply by pavelstianko** [reply]
@pavlanovotna already posted our template there, so take a look.

**Comment by pavlanovotna** [comment]
Hi, here's the full repository https://github.com/pavlanovotna/i18n-template. Still needs some work on language loading (see To do: in README)

**Reply by ansbr** [reply]
Thank you very much, I'll go and study the work you have done! 👍

**Reply by pavelstianko** [reply]
Also, if you find out how to get dafault language code and list of all languages we can make the template more universal :)

**Comment by marksteven** [comment]
Thankyou. This helps as I was not sure of the (correct) way to add a second page plus nested pages using i18n as / +page.svelte may not have been correct.




</p>
</details> +page.svelte etc may not have right

**Reply by pavelstianko** [reply]
You can just create subfolder with +page.svelte (like /about/+page.svelte or /about/test/+page.svelte)
